### ISSUE TITLE
Single Stats sets 100 as Max Data Points after upgrade to 4.0 beta1

### ISSUE THREAD

**Issue by AGirin** [grafana/grafana#6609]
- What Grafana version are you using? : 4.0 beta1
- What datasource are you using? : Graphite
- What did you do? : Upgraded from 3.x to 4.0 beta1

- What was the expected result?
Expected dashboards to keep settings.
Now every single stat panel sets Max Data point to 100:

![image](https://cloud.githubusercontent.com/assets/5177657/20364645/76579068-ac09-11e6-9e30-cabd59de14a4.png)

I need to modify all of them to set it to auto which is a little tedious since we have hundreds of dashboards.

**Comment by torkelo** [comment]
There is no change here, singlestat has had `maxDataPoints: 100` as default since it was introduced in v1.9. 

The reason for the default is that you seldom want more (usually you want less than 100), if you do not use the sparklines option you actually only need 1 data point. 

If you remove the `100` value and let it auto calculate the number of points Grafana will look at the width of the graph and set maxDataPoints to width of the panel in pixels, which will likely be over 100.

**Comment by AGirin** [comment]
Well - why then in the previous version all single stats panel were showing `auto` and not `100`? I did not even know about that 100 limit until I read your answer. I always assumed - it says auto then it is auto and not 100. 

So the displaying of this value changed - it was always Auto in any version I used before. Not sure if this is unique to our installation but we have two servers with grafana and they show the same behavior.
